packages/styles/variables.css

```diff
@@ -109,6 +109,7 @@
 
   /* icon sizes */
   --icon-size: 24px;
+  --tag-button-icon-size: 15px;
```

Comment:
To clarify, this is the right size but needs to be moved into the `tag-button.css` file, only global vars should exist in `variables.css`:

```css
:root {
  --tag-button-icon-size: 15px;
}
```